### ISSUE TITLE
broken link to normalization values

### ISSUE THREAD

**Issue by willtryagain** [pytorch/tutorials#2097]
## 📚 Documentation

There is a broken [link](https://pytorch.org/docs/master/torchvision/models.html) in the Load Data section of [this](https://pytorch.org/tutorials/beginner/finetuning_torchvision_models_tutorial.html) page.

cc @svekars @carljparker @datumbox @nairbv @fmassa @NicolasHug @YosuaMichael

**Comment by BaliDataMan** [comment]
Hey @svekars  Could you please confirm if https://pytorch.org/vision/0.14/models.html is correct replacement of broken link?

**Comment by svekars** [comment]
We no longer have a source file for this tutorial as it was likely removed or renamed. We should create an .rst file with a redirect like in [this example](https://github.com/pytorch/tutorials/blob/main/beginner_source/audio_data_augmentation_tutorial.rst). Redirect should point to: https://pytorch.org/tutorials/intermediate/torchvision_tutorial.html
